Sources/Datadog/Traces/DDSpan.swift

```diff
@@ -9,14 +9,16 @@ import Foundation
 
 internal class DDSpan: Span {
     private(set) var operationName: String
-    /// The `Tracer` which created this span.
-    internal let issuingTracer: DDTracer
+    private(set) var tags: [String: Codable]
```

Comment:
why is `tags` a `Dictionary` instead of an `Array`?
(i guess this is part of OT spec and i assume this is fine. i'm just wondering)

Reply:
[Open Tracing defines](https://github.com/opentracing/specification/blob/master/specification.md#set-a-span-tag) tags as key-value pairs:
> Required parameters
> * The tag key, which must be a string
> * The tag value, which must be either a string, a boolean value, or a numeric type

and because of this dictionary is more efficient than array. To me those are rather "attributes" than "tags", but OT standard needs to be language/technology agnostic, that's why "tags" is more general lingo I thing.

And yes, `[String: Codable]` is [enforced by `opentracing-swift`](https://github.com/opentracing/opentracing-swift/blob/e2b0551de2fd9a5cd9a3d8dff5390440f724283c/Sources/OpenTracing/Span.swift#L21).